contrib/functions/go/delete-annotations/README.md

```diff
@@ -0,0 +1,34 @@
+# delete-annotations
+
+## Overview
+
+<!--mdtogo:Short-->
+
+Deletes the supplied annotation keys from the resource(s) in a package. It can delete multiple annotations provided as a comma separated string as part of the function config (usage instructions below).
+
+<!--mdtogo:Long-->
+
+## Usage
+
+The function will execute as follows:
+
+1. Searches for resources with valid metadata
+2. Deletes the annotation keys provided in KptFile or imperatively as config parameters
+
```

Comment:
Please follow https://raw.githubusercontent.com/GoogleContainerTools/kpt-functions-catalog/master/functions/go/_template/README.md to ensure you have a `FunctionConfig` section.

Reply:
Acked